app/src/frontend/building/data-containers/location.tsx

```diff
@@ -96,7 +98,7 @@ const LocationView = (props) => (
             mode={props.mode}
             copy={props.copy}
             step={0.0001}
-            placeholder={51}
+            placeholder="51"
```

Comment:
The placeholder isn't really related to the type of the input value, it can any text to display in the input box so makes more sense to make it a string